testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -2818,6 +2877,32 @@ QUnit.module('datebox with time component', {
         assert.equal($clock.length, 0, 'clock was not rendered');
     });
 
+    QUnit.test('date box should chenge behavior if showAnalogClock option is changed at runtime', function(assert) {
+        const $element = $('#dateBox').dxDateBox({
+            type: 'datetime',
+            pickerType: 'calendar',
+            showAnalogClock: false,
+            opened: true
+        });
+        const instance = $element.dxDateBox('instance');
+
+        let $content = $(instance._popup.$content());
```

Comment:
Looks like we already have such test:
https://github.com/DevExpress/DevExtreme/blob/b56479ce50edf5f3423dea097f64e3e86ee0beae/testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js#L2800-L2819

Reply:
Here we check option changing on initialization and change the option value from false to true at runtime

Reply:
What about reworking tests these 2 tests to a few separate ones:
- check rendering when showAnalogClock = false initially
- check rendering when showAnalogClock = true initially
- check changing showAnalogClock from false to true
- check changing showAnalogClock from false to true

?